extensions/virtuallistview/src/VirtualListCell.tsx

```diff
@@ -223,6 +225,13 @@ export class VirtualListCell extends RX.Component<VirtualListCellProps, null> {
         return this._isVisible;
     }
 
+    setIsOnScreen(isOnScreen: boolean) {
```

Comment:
I don't like exposing the notion of "visible in view port" at the list cell level. A list cell shouldn't have this knowledge. The visible range is something that's maintained by the list, so if we add a callback, it seems like it should be exposed by the VirtualListView, not the VirtualListCell.